libs/platform/src/lib/components/split-menu-button/split-menu-button.component.ts

```diff
@@ -14,11 +14,14 @@ import { MenuComponent } from './../menu/menu.component';
 import { BaseComponent } from '../base';
 
 /**
- * Split Menu Button implementation based on the
- * https://github.com/SAP/fundamental-ngx/wiki/Platform:-Split-Menu-Button-Component-V1.0-Technical-Design
- * documents.
- *
+ * <fdp-split-menu-button [menu]="menu" [buttonLabel]="Default button" (primaryButtonClick)="onPrimaryButtonClick1()">
```

Comment:
Do we need these changes?

Reply:
Earlier we used to have tech doc in github wiki. but that practise is absolute now. so removed the reference.